deploy/examples/common.yaml

```diff
@@ -52,6 +52,18 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
No changes in this block, it just moved, right?

Reply:
Right.